### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by ghost**
In this branch I was working on cleaning the code; more than half of the work has been done.
But I do not think to leave there, I want to expand this library to the maximum adding more functionalities that would be very useful not only to learn but to add it to a real game.

### CODE REVIEW

**Comment by Shiro-** [comment]
Hi Reynardz,

Sorry for the late message.

I just wanted to say I have no plan on updating this repository. This is because of a few reasons. This repository was made following an A* Pathfinding tutorial. In the past I had worked with this algorithm before but I needed some sort of refresher.

Here is the tutorial in question: https://www.linkedin.com/learning/unity-5-2d-pathfinding

Now, while I followed this tutorial, I pushed it to this repository as a "remind me" of the basic concepts of the algorithm. And since I'm not currently working on pathfinding, I never thought of updating it.

In short, it is here as a refresher, or a learning tool for the basic concepts of the algorithm and nothing more. If you want more information I suggest you look at the tutorial link i posted with this message.

Here it is one more time: https://www.linkedin.com/learning/unity-5-2d-pathfinding

Sorry if this causes any trouble for you.

Happy Coding!

- Noel

**Comment by ghost** [comment]
No problems bro, but thanks so much for this repository
it help me a lot!.
